packages/amplify-cli/src/init-steps/s0-analyzeProject.ts

```diff
@@ -23,13 +25,64 @@ export async function analyzeProjectHeadless(context: $TSContext) {
   }
 }
 
+export function displayConfigurationDefaults(context, defaultProjectName, defaultEnv, defaultEditorName) {
```

Comment:
nit: add types for function parameters, here and elsewhere for new functions in .ts files